[Resource] Instance Refactor (#110)
powervs-ibm/terraform-provider-ibm: website/docs/r/pi_instance.html.markdown

The `ibm_pi_instance` provides the following [timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:

> * **create** - The creation of the instance is considered failed if no response is received for 120 minutes.

Reviewer: ```suggestion
* **create** - The creation of the instance is considered to have failed if no response is received for 120 minutes.
```

> * **Update** The updation of the instance is considered failed if no response is received for 60 minutes.

Reviewer: ```suggestion
* **update** The update of the instance is considered to have failed if no response is received for 60 minutes.```

> * **delete** - The deletion of the instance is considered failed if no response is received for 60 minutes.

Reviewer: ```suggestion
* **delete** - The deletion of the instance is considered to have failed if no response is received for 60 minutes.
```
